extract_endpoint/src/extract_endpoint/endpoint_runner.py

```diff
@@ -0,0 +1,13 @@
+import typing
+from extract_endpoint.endpoint_triggers import EndpointTrigger
+
+
+class EndpointRunner:
+    def __init__(self, triggers: typing.List[EndpointTrigger]) -> None:
+        self.triggers = triggers
+        return None
```

Comment:
🔥 Putting no return statement will result in a return value of `None`

Reply:
Oh duh lol, what was past me doing. Thanks for the catch